chrome/content/zotero/fileInterface.js

```diff
@@ -510,14 +510,17 @@ var Zotero_File_Interface = new function() {
 			format = "rtf";
 		}
 		
+		// determine locale preference
+		var locale = io.locale;
+		
 		// generate bibliography
 		try {
 			if(io.method == 'copy-to-clipboard') {
-				copyItemsToClipboard(items, io.style, false, io.mode === "citations");
+				copyItemsToClipboard(items, io.style, locale, false, io.mode === "citations");
```

Comment:
Oh, `this.` won't work there, actually, since the function it's in `_doBibliographyOptions()` is private itself. So in that case it should just use `Zotero_File_Interface.copyItemsToClipboard`.